src/run.js

```diff
@@ -485,15 +485,29 @@ const minimalcss = async options => {
     );
   }
 
+  // This is the protection against ever looking up the same CSS selector
+  // more than once. We can confidently pre-populate it with a couple that
+  // we're confident about.
+  const decisionsCache = { '*': true, body: true, html: true, '': true };
```

Comment:
What empty string selector `''` stands for?

Reply:
I guess there is an error where we convert AST to selectors, it should be `:after` instead of ``. Or at least we can add comment

Reply:
I think it comes from [`reduceCSSSelector`](https://github.com/peterbe/minimalcss/blob/7b2ee75c8a7446239ce969faac8bad20c28040ec/src/utils.js#L13). In terms of CSS selector and DOM inspecting the thing to look for is before the `:`. But if the whole CSS selector is `:after { ... }` then his is what happens.

Reply:
Bummer! I forgot this before I merged.

Reply:
Done https://github.com/peterbe/minimalcss/commit/d16f2c15b27eae1dd5cf1b2f18b71899f483e656